src/Storages/MergeTree/MergeTreeSettings.h

```diff
@@ -168,7 +168,7 @@ struct Settings;
     /** Settings for Stream */ \
     M(UInt64, shards, 1, "Shards number for Stream", 0) \
     M(UInt64, replicas, 1, "Replicas number for Stream", 0) \
-    M(String, sharding_expr, "rand()", "Sharding method of Stream. Default is 'rand()'.", 0) \
+    M(String, sharding_expr, "", "Sharding method of Stream. Default is 'rand()', or 'sip_hash64(<primary_keys...>)' if has primary keys.", 0) \
```

Comment:
change to “Sharding method of Stream. Empty string means ‘rand（）’ or ’sip_hash64() if primary key set‘